### DISCUSSION TITLE
Need help with activity-browser

### DISCUSSION THREAD

**Discussion by niassmatar96** [LCA-ActivityBrowser/activity-browser#1417] [Q&A]
Hello everyone,

I'm new to activity-browser and use the ecoinvent database. Do any of you have any idea how to extract the amount of material from each component of a technology. In other words, how can I extract from activity-browser the amount of material used in each process of the technology?
Thanks in advance

___
Original message:

>Bonjour à tous,
>
>Je suis nouveau sur activity-browser et j'utilise la base de données ecoinvent . Est-ce qu'un d'entre vous a une idée de comment >extraire la quantité de matériau de chaque composant d'une technologie. En d'autre terme comment extraire à partir d'activity->browser la quantité de matériau utilisé dans chaque processus de la technologie?
>Merci d'avance

**Comment by mrvisscher** [comment]
Please ask your questions in English

**Comment by niassmatar96** [comment]
Hello everyone,

I'm new to activity-browser and use the ecoinvent database. Do any of you have any idea how to extract the amount of material from each component of a technology. In other words, how can I extract from activity-browser the amount of material used in each process of the technology?
Thanks in advance

**Comment by marc-vdm** [comment]
Hi @niassmatar96 I'm not sure I understand your question correctly:

> how can I extract from activity-browser the amount of material used in each process of the technology?

What do you mean here with 'material' and 'technology'? I think it would help if you use [ISO or Brightway terms](https://github.com/LCA-ActivityBrowser/activity-browser/wiki/Getting-Started#understanding-activity-browser-terms) to make your question more clear.
